frontend/src/app/event/event-details/event-details.component.ts

```diff
@@ -63,6 +66,9 @@ export class EventDetailsComponent {
     };
     this.profile = data.profile;
     this.event = data.event;
+    this.eventDescriptionHTML = DOMPurify.sanitize(
+      marked.parse(this.event.description) as string
+    );
```

Comment:
I realize, this might be a good first use case where the Angular Directive construction might be useful. I wonder if we should give it a shot. The directive can then be applied to any Angular text box like this one and can then easily expand markdown support.

Here is some [documentation](https://angular.io/guide/attribute-directives) about directives. They are super useful, because we could end up with something like:

```
<div markdown>{{ input }}</div>
```

We might be expanding our usage of directives in the future, but it is a good place to start.